docs/resources/project.md

```diff
@@ -26,18 +26,25 @@ resource "couchbase-capella_project" "new_project" {
 ### Required
 
 - `name` (String) The name of the project (up to 128 characters).
+
+**Constraints:**
+  - Maximum length: 128 characters
 - `organization_id` (String) The GUID4 ID of the organization.
 
 ### Optional
 
 - `description` (String) A short description of the project (up to 256 characters).
+
+**Constraints:**
+  - Maximum length: 256 characters
 - `if_match` (String) A precondition header that specifies the entity tag of a resource.
```

Comment:
a bit of nit.  the way constraints render makes it confusing for the reader.  

<img width="1357" height="898" alt="Image" src="https://github.com/user-attachments/assets/f67716e4-6a37-41b0-b406-8284bbc1ab83" />

Reply:
can you elaborate on what seems confusing to the reader? so I will know what aspect to fix, thanks